openest/test/generate/test_functions.py

```diff
@@ -92,6 +92,85 @@ def test_spaninstabase():
         if yearresult[0] == 2:
             np.testing.assert_equal(yearresult[1], 1.)
 
+class TestFractionSum:
+    """
+    Basic tests for openest.generate.functions.FractionSum
+    """
+    @pytest.mark.parametrize(
+        "unshift_flag,n_expected",
+        [(True, 3), (False, 1)],
+        ids=['shifted', 'unshifted'],
+    )
+    def test_units_append(self, unshift_flag, n_expected):
+        """Tests that FractionSum correctly appends units from the original subcalcs
+        """
+        unit = ['fakeunit']
+        frac_mock = MockAppCalc(years=[0], values=[0.1], unitses=unit)
```

Comment:
Shouldn't have `fakeunit`.

Reply:
What do we use to indicate unitless-ness? Calculations require some form of input for `unitses`, so just a list of `None` (this is how it's now done in 
acc586a).

Reply:
Resolved in ec0e39203139fd0632a354e398e6075aea9f8cfe. We're using "unitless".